sdk/dotnet/Pulumi/PublicAPI.Shipped.txt

```diff
@@ -178,11 +178,9 @@ Pulumi.ResourceTransformationArgs
 Pulumi.ResourceTransformationArgs.Args.get -> Pulumi.ResourceArgs
 Pulumi.ResourceTransformationArgs.Options.get -> Pulumi.ResourceOptions
 Pulumi.ResourceTransformationArgs.Resource.get -> Pulumi.Resource
-Pulumi.ResourceTransformationArgs.ResourceTransformationArgs(Pulumi.Resource resource, Pulumi.ResourceArgs args, Pulumi.ResourceOptions options) -> void
```

Comment:
We can't take this breaking change in a minor version. Could we provide a new constructor overload instead?